scripts/evaluate.py

```diff
@@ -10,7 +10,7 @@
 os.environ["TF_CPP_MIN_LOG_LEVEL"] = "2"
 
 from doctr.utils.metrics import LocalizationConfusion, ExactMatch, OCRMetric
-from doctr.datasets import FUNSD
+from doctr.datasets import FUNSD, SROIE
```

Comment:
let's avoid importing it since we don't use it for now

Reply:
ok